transducer.sh

```diff
@@ -0,0 +1,28 @@
+yoyodyne-train --model_dir=../ \
```

Comment:
you probably didn't mean to check this in?

Reply:
no. no i did not. 

minor rant: have you noticed that